components/prism-orfeo.js

```diff
@@ -0,0 +1,89 @@
+(function (Prism) {
+	Prism.languages.orfeo = {
+		'parenthesis-open': {
+			pattern: /(^|[(^\s])\((?=[^.]|$)/,
+			alias: 'punctuation',
+			lookbehind: true
+		},
+		'parenthesis-close': {
+			pattern: /(^|[^^])\)(?=[).\s]|$)/,
+			alias: 'punctuation',
+			lookbehind: true
+		},
+		'duration': {
+			pattern: /(^|[(\s]):(?:=|(?:\.\d+|(?:0|[1-9]\d*)(?:\.\d*)?)(?:\/(?:0?\.0*[1-9]\d*|[1-9]\d*(?:\.\d*)?))?)(?=[)\s]|$)/,
+			alias: 'number',
+			lookbehind: true
+		},
+		'datum-quoted': {
+			pattern: /(^|[(^\s])"[^"]*"(?=[).\s]|$)/,
+			alias: 'string',
+			lookbehind: true
+		},
+		'datum-unquoted': {
+			pattern: /(^|[(^\s])[.\w/\[\]♮♭♯-]*[\w/\[\]♮♭♯-](?=[).\s]|$)/,
+			alias: 'string',
+			lookbehind: true
+		},
+		'rhythm-flags': {
+			pattern: /(^|[(\s])\^+(?=\(|"[^"]*"|[.\w/\[\]♮♭♯-]*[\w/\[\]♮♭♯-]|$)/,
+			alias: 'operator',
+			lookbehind: true
+		},
+		'rhythm-dots': {
+			pattern: /(^|[)"\w/\[\]♮♭♯-])\.+(?=[)\s]|$)/,
+			alias: 'operator',
+			lookbehind: true
+		},
+	}
+}(Prism))
+/* -----------------------------------------------------------------------------
+	Orfeo v0.1.0
```

Comment:
Hi,

About the `^^^"datum"` issue, strange... I will test it again on my end. I must have been doing something wrong.

Don't be sorry.  No worries. I am planning on making a website with full docs. This should be ready soon - it's what I am working on now. The reason I did a syntax highlighter in Prism is for that very purpose since the docs will use it and be done with docsify. I can generate the Prism files myself for now and use them to make the docs, then once you see the docs etc... you can merge this commit into Prism. I have EBNF files and details about the language etc... usage examples, the whole works in those docs. So, let's wait for me to finish the docs, then I'll give you a heads up here and you'll have a much better idea of the entire language. They should be done sometime in November.

`(btw. the link to the Orfeo organization in your profile readme is incorrect)`

Thanks for the heads up. I changed the organization name and forgot to reflect that in my README.md

Reply:
Thank you. Then I'll be waiting until the docs are up.

If the `^^^"datum` issue is still there, then we can talk about it. Feel free to ask anytime.